example/RNExampleApp/config/TINDOTConfig.js

```diff
@@ -1,5 +1,4 @@
 export default {
-    license: '', //INSERT YOUR LICENSE KEY HERE
```

Comment:
@vimalvenugopalananyline was this removed on purpose? I would recommend adding this line back

Reply:
had removed it.. added back on @igor-graca request.

Reply:
@vimalvenugopalananyline @igor-graca got it, thanks for the explanation! You have my approval, @igor-graca can you give yours too when you think it's ready to merge 🙏 Thanks both!